conftest.py

```diff
@@ -242,6 +252,42 @@ def get_driver(self):
         return Chrome(options=options)
 
 
+@pytest.hookimpl(hookwrapper=True)
+def pytest_runtest_call(item):
+    """We want to run extra verification at the start and end of each test to
+    check that we haven't leaked memory. According to pytest issue #5044, it's
+    not possible to "Fail" a test from a fixture (no matter what you do, pytest
+    sets the test status to "Error"). The approach suggested there is hook
+    pytest_runtest_call as we do here. To get access to the selenium fixture, we
+    immitate the definition of pytest_pyfunc_call:
+    https://github.com/pytest-dev/pytest/blob/6.2.2/src/_pytest/python.py#L177
+
+    Pytest issue #5044:
+    https://github.com/pytest-dev/pytest/issues/5044
+    """
+    selenium = None
+    if "selenium" in item._fixtureinfo.argnames:
+        selenium = item.funcargs["selenium"]
+    if "selenium_standalone" in item._fixtureinfo.argnames:
+        selenium = item.funcargs["selenium_standalone"]
+    if selenium and pytest.mark.norefs.mark not in item.own_markers:
```

Comment:
```suggestion
    if selenium and pytest.mark.skip_refcount_check.mark not in item.own_markers:
```
to be more explicit.